.travis.yml

```diff
@@ -21,13 +21,16 @@ cache:
         - $HOME/.composer/cache
 
 before_install:
+    - mv $HOME/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini $HOME/xdebug.ini || return 0
     - 'if [ "$SYMFONY_VERSION" != "" ]; then sed -i "s/\"symfony\/\([^\"]*\)\": \"[^\"]*\"/\"symfony\/\1\": \"$SYMFONY_VERSION\"/g" composer.json; fi'
 
 install:
+    - travis_retry composer global require hirak/prestissimo
```

Comment:
This actually ends up slower surely? It takes just as long to install this globally as it does to install the rest.